l10n/configs/pontoon.toml

```diff
@@ -217,6 +217,13 @@ locales = [
 [[paths]]
     reference = "en/firefox/whatsnew/whatsnew.ftl"
     l10n = "{locale}/firefox/whatsnew/whatsnew.ftl"
+[[paths]]
+    reference = "en/firefox/whatsnew/whatsnew-fx98-vpn-eu.html"
+    l10n = "{locale}/firefox/whatsnew/whatsnew-fx98-vpn-eu.html"
+    locales = [
```

Comment:
**question (non-blocking, curiousity):** what's the best way to confirm the file will show up for these locales in pontoon? checking they are part of the [PROD_LANGUAGES settings](https://github.com/mozilla/bedrock/blob/master/bedrock/settings/base.py#L129)?

Reply:
`PROD_LANGUAGES` is a list of all the supported language codes we use in production, but this specific `locales` list here is what gets used to ensure only a subset of those locales appears in Pontoon.